server/config.js

```diff
@@ -36,6 +36,11 @@ const conf = convict({
     format: ['production', 'development', 'test'],
     default: 'development',
     env: 'NODE_ENV'
+  },
+  max_file_size: {
+    format: Number,
+    default: (1024 * 1024 * 1024) * 2,
```

Comment:
Fair enough. But to be clear, the UI says the recommended file limit is 1GB for uploads, but the server will allow up to ~2GB max?

<img width="390" alt="firefox_send" src="https://user-images.githubusercontent.com/557895/28444294-03c1eea2-6d71-11e7-9395-682d078fbfd8.png">